docs/configuration/coordinates.rst

```diff
@@ -0,0 +1,136 @@
+Coordinates in Navit
+====================
+
+Various parts of Navit will read geographical coordinates provided as
+text:
+
+-   the `textfile <https://wiki.navit-project.org/index.php/Textfile>`__
+   map format
+
+-   the "center=" attribute in the configuration file
+
+-   some Navit commands (e.g. set_position), which can be invoked via
+   the `internal
+   GUI <https://wiki.navit-project.org/index.php/Internal_GUI>`__ or the
+   `Dbus <https://wiki.navit-project.org/index.php/Dbus>`__ bindings
+
+-   the files for bookmarks and last map position (bookmarks.txt and
+   center.txt)
+
+This page documents the coordinate systems and formats that Navit will
+accept.
+
+Supported coordinate systems and formats
+========================================
+
+.. _longitude--latitude-in-decimal-degrees:
+
+Longitude / Latitude in decimal degrees
+---------------------------------------
+
+Longitude / latitude in degrees can be specified as signed decimal
+fractions:
+
+.. code:: 
+
+   -33.3553 6.334
+
+That would be about 33° West, 6° North. Note that in this format
+longitude comes first. The coordinates are assumed to be based on WGS84
+(the coordinate system used by the GPS system, and by practically all
+common navigation systems).
+
+.. _latitude--longitude-in-degrees-and-minutes:
+
+Latitude / Longitude in degrees and minutes
+-------------------------------------------
+
+Latitude / Longitude can also be specified in degress and minutes with
+compass directions (N/S, E/W):
+
+.. code:: 
+
+   4808 N 1134 E
+
+Latitude and longitude are multiplied by 100, so the position above
+corresponds to 48°8' N, 11°34' (Munich).
+
+For greater precision you can write the minutes as decimal fractions:
+
+.. code:: 
+
+   4808.2356 N 1134.5252 E
+
+That is 48°8.2356' N 11°34.5252' E, the center of the Marienplatz in
+Munich.
+
+Notes:
+
+-   This format is rather unusual (because it uses arcminutes, but not
+   arcseconds). It is probably easier to just use decimal fractions of
+   degrees.
+
+-   The spaces are relevant for parsing. Use exactly one space between
+   the number and the letter N/S/E/W.
+
+Cartesian coordinates
+---------------------
+
+Internally, Navit uses a cartesian coordinate system induced by a
+Mercator projection. Coordinates are written as hexadecimal integers:
+
+.. code:: 
+
+   0x13a3d7 0x5d6d6d
+
+or specifying a projection:
+
+.. code:: 
+
+   mg: 0x13a3d7 0x5d6d6d
+
+That is again 48°8.2356' N 11°34.5252' E. The part up to and including
+the colon is optional, it names the projection to use. Possible values:
+
+-   mg - the projection used by Map&Guide (the default)
+
+-   garmin - "Garmin" projection (TODO: When would it be useful?)
+
+This format is used internally by Navit, but is probably not very useful
+for other purposes.
+
+Google Maps Format
+------------------
+
+.. code:: 
+
+   48.137260, 11.575420
```

Comment:
Since there is another WGS84 decimal format, albeit with longitude before latitude, how does Navit know which of the two is meant (i.e. order of coordinates)? Or, what do I need to pay attention to when supplying these coordinates so Navit interprets them correctly (avoiding coordinates being swapped)?